build/jobs/deploy-aks.yml

```diff
@@ -0,0 +1,101 @@
+
+parameters:
+- name: version
+  type: string
+- name: dataStore
+  type: string
+- name: deploymentEnvironmentName
```

Comment:
Nit: We haven't been passing deployment environment name as a parameter because the global variable DeploymentEnvironmentName can be used.